server/routes/overview/mappers/prisonerSupportNeedsMapper.ts

```diff
@@ -1,30 +1,27 @@
 import { parseISO, startOfDay } from 'date-fns'
-import type { HealthAndSupportNeeds, PrisonerSupportNeeds } from 'viewModels'
-import type { LearnerProfile } from 'curiousApiClient'
+import type { LddAssessment, PrisonerSupportNeeds } from 'viewModels'
+import type { AllAssessmentDTO, LearnerLatestAssessmentV1DTO, LearnerLddInfoExternalV1DTO } from 'curiousApiClient'
 
-const toPrisonerSupportNeeds = (
-  learnerProfiles: Array<LearnerProfile>,
-  prisonNamesById: Record<string, string>,
```

Comment:
Prison name is no longer mapped in the mapper, so no need for it to be passed the prison data